components/section/Meetups.vue

```diff
@@ -111,6 +111,16 @@ export default {
                         long as you are kind, supportive, and respectful of
                         others.`,
                 },
+                dataScience: {
+                    name: "Edmonton Data Science Community",
+                    logoSvg: "https://ik.imagekit.io/des/meetups/DataScience_Nvpcx6-aY?updatedAt=1698789157558",
```

Comment:
you should be able to just do a relative path like on like 90. Also note that this is an image, not an svg